Dockerfile

```diff
@@ -1,4 +1,4 @@
-from oven/bun:1.0-slim
+from oven/bun:1.0.30-slim
```

Comment:
bug in bun's newest docker image broke the build (perhaps related to this [issue](https://github.com/oven-sh/bun/issues/9428))